src/tools/pub/package-layout.md

```diff
@@ -58,14 +58,11 @@ enchilada/
     style.css
 {% endprettify %}
 
-\* The `.dart_tool/` directory exists after you've run `pub get`. 
```

Comment:
Running `pub get` creates the `.dart_tool` directory if none exists.
To learn more, 
see [Project specific caching for tools](#project-specific-caching-for-tools).

Reply:
I've manually pushed a fix adding this back.